website/docs/api/classes/TodoistApi.md

```diff
@@ -91,7 +91,54 @@ A promise that resolves to the created label.
 ### addProject()
 
 ```ts
-addProject(args: AddProjectArgs, requestId?: string): Promise<Project>
+addProject(args: AddProjectArgs, requestId?: string): Promise<
+  | {
+  canAssignTasks: boolean;
+  childOrder: number;
+  color: string;
+  createdAt: null | string;
+  defaultOrder: number;
+  description: string;
+  id: string;
+  inboxProject: boolean;
+  isArchived: boolean;
+  isCollapsed: boolean;
+  isDeleted: boolean;
+  isFavorite: boolean;
+  isFrozen: boolean;
+  isShared: boolean;
+  name: string;
+  parentId: null | string;
+  updatedAt: null | string;
+  url: string;
+  viewStyle: string;
+ }
+  | {
+  canAssignTasks: boolean;
+  childOrder: number;
+  collaboratorRoleDefault: string;
+  color: string;
+  createdAt: null | string;
+  defaultOrder: number;
+  description: string;
+  folderId: null | boolean;
+  id: string;
+  isArchived: boolean;
+  isCollapsed: boolean;
+  isDeleted: boolean;
+  isFavorite: boolean;
+  isFrozen: boolean;
+  isInviteOnly: null | boolean;
+  isLinkSharingEnabled: boolean;
+  isShared: boolean;
+  name: string;
+  role: null | string;
+  status: string;
+  updatedAt: null | string;
+  url: string;
+  viewStyle: string;
+  workspaceId: string;
+}>
```

Comment:
Can this return type not look so unwieldly? Shouldn't this just be `PersonalProject | WorkspaceProject`? This applies to all the added docs in this PR.

Reply:
This is something that I had to battle with when I first set up docusaurus/typedoc. Types are automatically expanded, while interfaces aren't. Because `Project` can only be a type because it is a union of two interfaces, it appears expanded.

I've now revisited typedoc/docusaurus documentation looking for some config where we can disable auto expanding but once again I haven't found one.

That said, as a solution we can use directly the `PersonalProject` and `WorkspaceProject` interfaces. In fact, I think it's better than having a `Project` union as it makes the project distinction very explicit.

Done in https://github.com/Doist/todoist-api-typescript/pull/286/commits/a922ba64ecba0e423802aa09d7633948a9876597 .